frontend/src/__tests__/cypress/cypress/utils/oc_commands/baseCommands.ts

```diff
@@ -0,0 +1,21 @@
+import type { CommandLineResult } from '~/__tests__/cypress/cypress/types';
+
+/**
+ * Applies the given YAML content using the `oc apply` command.
+ *
+ * @param yamlContent YAML content to be applied
+ * @returns Cypress Chainable
+ */
+export const applyOpenShiftYaml = (yamlContent: string): Cypress.Chainable<CommandLineResult> => {
+  const ocCommand = `oc apply -f - <<EOF\n${yamlContent}EOF`;
```

Comment:
Does this need an extra `\n` to be safe?
```suggestion
  const ocCommand = `oc apply -f - <<EOF\n${yamlContent}\nEOF`;
```